R/input-numeric.R

```diff
@@ -35,7 +35,7 @@ numericInput <- function(inputId, label, value, min = NA, max = NA, step = NA,
   value <- restoreInput(id = inputId, default = value)
 
   # build input tag
-  inputTag <- tags$input(id = inputId, type = "number", class="form-control",
+  inputTag <- tags$input(id = inputId, type = "number", class="form-control shiny-input-number",
```

Comment:
For consistency with other `form-control` usage

```suggestion
  inputTag <- tags$input(id = inputId, type = "number", class="shiny-input-number form-control",
```